doc/source/data/accessing-datasets.rst

```diff
@@ -1,10 +1,10 @@
-.. _exchanging_datasets:
+.. _accessing_datasets:
 
 ===================
-Exchanging Datasets
+Accessing Datasets
 ===================
 
-Datasets can be passed to Ray tasks or actors and read with ``.iter_batches()`` or ``.iter_rows()``.
+Datasets can be accessed by Ray tasks or actors and read with ``.iter_batches()`` or ``.iter_rows()``.
```

Comment:
The fact that Datasets can be passed to Ray tasks and actors without incurring data movement is an important feature, so I don't know if we should change this language. And here, "accessed" and "read" seem to mean the same thing.

Reply:
Polished.